### PR TITLE
Fix some ShellCheck reports

### PR DESCRIPTION
**Pull request by lzaoral**
See commit messages for details.

### CODE REVIEW

**Review by kdudka — CHANGES_REQUESTED** [review]
Some of the changes in this pull request broke the `infer` plug-in.  Tested with:
```
% curl -LO https://github.com/facebook/infer/releases/download/v1.1.0/infer-linux64-v1.1.0.tar.xz
% csmock -r fedora-37-x86_64 -t infer --infer-archive-path ./infer-linux64-v1.1.0.tar.xz ...
```

It fails with:
```
>>> 2023-02-01 17:05:35 "/usr/bin/mock" "-r" "fedora-37-x86_64" "--plugin-option=tmpfs:keep_mounted=True" "--config-opts=print_main_output=True" "--quiet" "--chroot" "/usr/share/csmock/scripts/install-infer.sh /home/kdudka/git/cswrap/infer-linux64-v1.1.0.tar.xz /builddir/infer-out"
NOTE: INFER: install-infer.sh: Infer archive extracted successfully
NOTE: INFER: install-infer.sh: Infer symlink created successfully
ERROR: INFER: install-infer.sh: Failed to run 'infer --version' to test a symlink to /opt//opt/infer-linux64-v1.1.0/bin/infer
```

Downgrade to stable version of the `infer` plug-in fixes the above error.

File: scripts/run-bandit.sh [hunk 1]
```diff
@@ -10,18 +10,18 @@ echo "[DEBUG]	SEC_LEVEL = $SEC_LEVEL" >&2
...
 
-# Find all python files and run bandit analysis 
+# Find all python files and run bandit analysis
 find "$@" -type f -print0 \
     | xargs -0 sh -c 'for i in "$@"; do { printf "%s\n" "$i" | grep "\\.py\$" \
         || head -n1 "$i" | grep --text "^#!.*python"
         } >/dev/null && readlink -f "$i"; done' "$0" \
     | sort -V | tee /dev/fd/2 \
-    | xargs -r bandit -n -1 -r $SEC_LEVEL --format custom \
+    | xargs -r bandit -n -1 -r "$SEC_LEVEL" --format custom \
```
[the remark this answers is not in the document]
Comment (lzaoral) [reply scripts/run-bandit.sh]: Ok, I've dropped the commit for now.

File: scripts/chroot-fixups/symbiotic-timeout.sh [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-for file in /opt/symbiotic/lib/symbioticpy/symbiotic/{transform,verifier}.py; do
+for filename in transform verifier; do
+    file="/opt/symbiotic/lib/symbioticpy/symbiotic/${filename}.py"
```
[the remark this answers is not in the document]
Comment (lzaoral) [reply scripts/chroot-fixups/symbiotic-timeout.sh]: Sure. Addressed in 6ff2de591806c236b4914c621efb50ad39c8b94d.

**Comment by lzaoral** [comment]
Thanks for the review! I forgot to test the infer change properly. Should be fixed in d7eca6e9d4db84259d1767f9110c595c6481d900.

**Review by kdudka — APPROVED** [review]
All looks good now.  Thanks for the update!

**Pull request merged**